process-dir.js

```diff
@@ -146,7 +147,15 @@ async function replacePackage (dir, file, files) {
   pkg.module = 'index.js'
   pkg['react-native'] = 'index.js'
 
-  if (files.includes(file.replace(/\.js$/, '.browser.js'))) {
+  if (
+    nodeEnvReplacementTargets.includes(file) ||
```

Comment:
Too long. Can we replaced it to `envTargets`?

Reply:
Sure. Does it increase package size or it's just personal preference?

Reply:
Just code style consistancy

Reply:
Shortened this and other names to be consistent